sdk/openai/openai/review/openai-types.api.md

```diff
@@ -45,13 +67,17 @@ export interface AzureCosmosDBChatExtensionConfiguration extends AzureChatExtens
 
 // @public
 export interface AzureCosmosDBChatExtensionParameters {
+    allow_partial_result?: boolean;
     authentication: OnYourDataAuthenticationOptions;
     container_name: string;
     database_name: string;
     embedding_dependency: OnYourDataVectorizationSource;
     fields_mapping: AzureCosmosDBFieldMappingOptions;
     in_scope?: boolean;
+    include_contexts?: OnYourDataContextProperty[];
     index_name: string;
+    max_search_queries?: number;
+    role_information?: string;
```

Comment:
`role_information` are not included in the definition in the inference file [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/preview/2024-10-01-preview/inference.yaml#L3093-L3145). In TypeSpec file [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/extensions/oyd/common_options.tsp#L28-L31), this property is removed since Aug API version, so I think it's not included in this API version
```suggestion
```